cabal-testsuite/src/Test/Cabal/OutputNormalizer.hs

```diff
@@ -27,6 +27,7 @@ normalizeOutput nenv =
     -- Install path frequently has architecture specific elements, so
     -- nub it out
   . resub "Installing (.+) in .+" "Installing \\1 in <PATH>"
+  . resub "  error, called at .*" "<CABAL_ERROR>"
```

Comment:
This change needs a comment just like most of the lines above and below. And probably a more expanded comment somewhere else (e.g. commit message and/or PR description). It looks fishy tbh (I'm happy to be convinced otherwise, but again, please provide an explanation for it). The root of the problem, I think, is in the code where you call `error`: I don't believe it should be done that way.

Reply:
This is now removed because the build does not produce an exception.